tools/all_syscalls

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
This breaks the build during CI.

Reply:
We have almost everywhere bash ...

Reply:
I'm aware, i only left the important utilities (small ones, ones that dont require bash all that much), but focused on the one that is only required at build-time, the rest aren't required at build-time.

Reply:
Unfortunately `set -o pipefail` is not POSIX and it breaks again on systems where `/bin/sh` is not bash.

Reply:
> Unfortunately set -o pipefail is not POSIX
```
# pipefail is part of modern POSIX
```

Reply:
It still breaks our build. And users *will* encounter this issue.

Reply:
I'm encountering this issue at https://github.com/util-linux/util-linux/pull/2394.

![image](https://github.com/util-linux/util-linux/assets/77077/a03298f1-a3f5-4e3f-a19d-7e7a152ac944)

Reply:
unfortunately there is no realistic or clean way to have a pipefail-like feature in a script like this, you either have to:
1. disable the building/requirement of the all_syscalls via an available option
2. update your shell to latest POSIX